Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -107,13 +107,14 @@ def uninstall_launchctl(*services, command: nil, **_)
         all_services.each do |service|
           ohai "Removing launchctl service #{service}"
           booleans.each do |with_sudo|
+            sudo_user = with_sudo ? "root" : ""
             plist_status = command.run(
               "/bin/launchctl",
               args: ["list", service],
-              sudo: with_sudo, print_stderr: false
+              sudo: sudo_user, print_stderr: false
             ).stdout
             if plist_status.start_with?("{")
-              command.run!("/bin/launchctl", args: ["remove", service], sudo: with_sudo)
+              command.run!("/bin/launchctl", args: ["remove", service], sudo: sudo_user)
```

Comment:
Is `with_sudo` still used anywhere?

Reply:
Only as a loop variable. Do you suggest mapping?

Reply:
No worries if it's still used.